### DISCUSSION TITLE
Dashboard's PR links break on GitHub Projects

### DISCUSSION THREAD

**Discussion by wopian** [renovatebot/renovate#21711] [Suggest an Idea]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please select which platform you are using if self-hosting.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Links to Renovate-managed PRs in the Dependency Dashboard use relative links instead of absolute links, this unfortunately means these PR links do not work when the issue is opened in a [GitHub Project](https://github.com/features/issues) as it tries to send you to an invalid path.

For example:

`[build: update angular to 14.1.1](../pull/148)`

When opened from a GitHub Project page (e.g `https://github.com/orgs/:org/projects/1/views/1`) it will send you to `https://github.com/orgs/:org/projects/1/pull/148` before redirecting to GitHub's 404 fallback.

Is Renovate able to know the absolute path of the repositories it opens PRs in to use the absolute paths here, so links in the Dependency Dashboard work wherever they're opened? (e.g issues embedded in Slack/other services that support rich embeds too)

We have all of the Dependency Dashboards of our services in a GitHub project for easier inspection to check everything is functioning correctly.

![image](https://user-images.githubusercontent.com/3440094/184130709-e9db0fef-ec26-474a-a063-9606dc9dda40.png)

### Have you created a minimal reproduction repository?

A public example of the above:

https://github.com/users/wopian/projects/2/views/1

**Comment by PhilipAbed** [comment]
Sounds to me like a feature more than a bug, this looks like a another platform that is derived from github platform,
maybe you should change the description to something like;
Support Relative Links on `Github Project`

**Comment by wopian** [comment]
> Sounds to me like a feature more than a bug

Yeah I thought about opening it as a feature request but I felt it was more suitable as a bug, as it is unintended behaviour of the issue links in the Dashboard's PRs list

> this looks like a another platform that is derived from github platform

It is GitHub.com, not a third party platform, unless I misunderstood this

**Comment by wopian** [comment]
Added a reproduction project using my public repos at https://github.com/users/wopian/projects/2/views/1

**Comment by rarkins** [comment]
I think we use relative links to save space and reduce complexity in our markdown, but if even github themselves mash them up when referencing them in Projects view then we may need to refactor to use full URLs.

**Comment by wopian** [comment]
Made the repro visible to public - didn't notice it defaulted to private 😰

**Comment by ghost** [comment]
I'm facing the same issue with relative links to PRs on the dashboard container within my organizations project.
The organization project is one of the best solutions to get an overview on all your renovate topics within an organization. We are having more than 50 repos in there to manage. And jumping through all of them with a list of links wherever maintained (wiki, excel,...) is cumbersome.
By solving this PR link issue, i am convinced, we would get a big step closer to a time solving solution.

**Comment by jamietanna** [comment]
Per https://github.com/renovatebot/renovate/discussions/38823 this is now tracked under https://github.com/renovatebot/renovate/issues/38853
